WebHostLib/upload.py

```diff
@@ -20,6 +20,43 @@
 
 banned_zip_contents = (".sfc", ".z64", ".n64", ".sms", ".gb")
 
+def parse_multidata(compressed_multidata, files={}):
```

Comment:
Last but not least: a function called `parse_*` writing to the database is not good. Maybe `process_multidata`?

Reply:
done

Reply:
did you forget to push?